src/map/options.ts

```diff
@@ -420,4 +420,18 @@ export class MapOptions extends OptionsGroup {
     public colorScale(): Plotly.ColorScale {
         return COLOR_MAPS[this.palette.value];
     }
+
+    public logLinearLabelSwitch = (axis: AxisOptions, axisName: string) => {
+        const optionsContainer = getByID<HTMLElement>(`chsp-extra-${axisName}`);
+        const minInputLabel = optionsContainer.getElementsByClassName('input-group-text')[1];
```

Comment:
this feels really fragile, and would break when we change the HTML layout. How about adding an id to the labels and getting the min/max labels this way?

Also, this would be perfect for a unit test!